src/app/pages/MarginTradePage/components/OpenPositionsTable/index.tsx

```diff
@@ -73,6 +77,19 @@ export function OpenPositionsTable(props: Props) {
               <td colSpan={99}>{t(translations.openPositionTable.noData)}</td>
             </tr>
           )}
+          {transactions.length > 0 && (
+            <>
+              {transactions
+                .filter(
+                  tx =>
+                    tx.type === TxType.TRADE &&
+                    [TxStatus.FAILED, TxStatus.PENDING].includes(tx.status),
+                )
```

Comment:
I think filter here will cause rerender if unrelated transaction will be added / updated to the transactions. Maybe filtering should be done in the memo?